src/main/java/io/hops/hopsworks/expat/migrations/airflow/DagsMigration.java

```diff
@@ -0,0 +1,88 @@
+package io.hops.hopsworks.expat.migrations.airflow;
+
+import io.hops.hopsworks.common.util.ProcessDescriptor;
+import io.hops.hopsworks.common.util.ProcessResult;
+import io.hops.hopsworks.expat.configuration.ConfigurationBuilder;
+import io.hops.hopsworks.expat.configuration.ExpatConf;
+import io.hops.hopsworks.expat.db.DbConnectionFactory;
+import io.hops.hopsworks.expat.executor.ProcessExecutor;
+import io.hops.hopsworks.expat.migrations.MigrateStep;
+import io.hops.hopsworks.expat.migrations.MigrationException;
+import io.hops.hopsworks.expat.migrations.RollbackException;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.configuration2.Configuration;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.concurrent.TimeUnit;
+
+public class DagsMigration implements MigrateStep {
+  private static final Logger LOGGER = LogManager.getLogger(DagsMigration.class);
+
+  private String expatPath = null;
+  private String hadoopHome = null;
+  private String hopsClientUser = null;
+
+  private void setup() throws ConfigurationException {
+    Configuration config = ConfigurationBuilder.getConfiguration();
+    expatPath = config.getString(ExpatConf.EXPAT_PATH);
+    hopsClientUser = config.getString(ExpatConf.HOPS_CLIENT_USER);
+    hadoopHome = System.getenv("HADOOP_HOME");
+  }
+
+  @Override
+  public void migrate() throws MigrationException {
+    try (Connection connection = DbConnectionFactory.getConnection();
+         Statement stmt = connection.createStatement();
+         ResultSet resultSet = stmt.executeQuery("SELECT projectname,users.username FROM project " +
+             "JOIN users ON project.username=users.email;")) {
+      setup();
+      while (resultSet.next()) {
+        String projectName = resultSet.getString("projectname");
+        Integer projectId = resultSet.getInt("id");
```

Comment:
The query selects the project name and the user name, here you are retrieveing the id from the results.